playbooks/group_vars/elasticsearch.yml

```diff
@@ -0,0 +1 @@
+acs_play_elasticsearch_major_version: 7.x
```

Comment:
once we have 25 out, should this be moved to version-dependant vars or not?

Reply:
only for acs25. Other versions would default here